### ISSUE TITLE
Error in mounted hook: "TypeError: Cannot read property 'onValueUpdate' of undefined"

### ISSUE THREAD

**Issue by akoidan** [jacobmischka/vue-flatpickr#2]
` <Flatpickr />` produces error below w/o vue-runtime.
```
vue.runtime.esm.js?a427:542 TypeError: Cannot read property 'onValueUpdate' of undefined
    at VueComponent.mounted (index.js?7e74:3)
    at callHook (vue.runtime.esm.js?a427:2642)
    at Object.insert (vue.runtime.esm.js?a427:3497)
    at invokeInsertHook (vue.runtime.esm.js?a427:5436)
    at Vue$3.patch [as __patch__] (vue.runtime.esm.js?a427:5606)
    at Vue$3.Vue._update (vue.runtime.esm.js?a427:2391)
    at Vue$3.updateComponent (vue.runtime.esm.js?a427:2515)
    at Watcher.get (vue.runtime.esm.js?a427:2858)
    at new Watcher (vue.runtime.esm.js?a427:2847)
    at mountComponent (vue.runtime.esm.js?a427:2519)

```

**Comment by jacobmischka** [comment]
I really need to go through and refactor all of the build config, this is all leftover from the original repo and it's very confusing. I'll try to get to it this weekend, thanks for reporting the issue.

**Comment by jacobmischka** [comment]
This should be fixed now in 3.0 I believe, please let me know if not.
